README.md

```diff
@@ -295,16 +392,96 @@ Here is an example:
 (jeison-read 'boolean json-company '(company CEOs 0 founder)) ;; => t
 ```
 
-Unlike arguments to `elt` function, indices can be *negative*. Negative indices have the same semantics as in **Python** language (enumeration from the end):
+Unlike arguments to `elt` function, indices can be *negative*. Negative
+indices have the same semantics as in **Python** language (enumeration from
+the end):
 
 ``` emacs-lisp
 (jeison-read 'string json-company '(company CEOs -1 name last)) ;; => "Johnson"
 (jeison-read 'boolean json-company '(company CEOs -2 cool)) ;; => t
 ```
 
+### Hash tables
+
+There are times when the JSON stream doesn't lend itself to lists or EIEIO
+objects. One such instance is the [CMake][cmake] file API's `reply` attribute:
+
+```json
+	"reply" :
+	{
+		"client-1b4f70c2f678bf78" :
+		{
+			"codemodel-v2" :
+			{
+				"jsonFile" : "codemodel-v2-5b0f688c981e1e250cef.json",
+				"kind" : "codemodel",
+				"version" :
+				{
+					"major" : 2,
+					"minor" : 4
+				}
+			}
+		}
+	}
+```
```

Comment:
In all previous examples I used spaces for indentation, so maybe here we can also do that?